sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -472,7 +470,7 @@ def get_tx_power(self):
         if tx_power_support is None:
             return None
         if not tx_power_support:
-            return ["N/A" for _ in range(self.NUM_CHANNELS)]
+            return {'OpticalPowerTx%dField' %i : "N/A" for i in range(1, self.NUM_CHANNELS+1)}
```

Comment:
This will fail sonic-mgmt test since the API currently expects a list.

Reply:
thanks. now fixed